solvcon/devenv
# Re-locate helper scripts building SOLVCON

This pull request is the follow-up implementation for the discussion https://github.com/solvcon/devenv/issues/13#issuecomment-820741629 . The namespace of the folder/file hierarchy is still open and welcome for more inputs, comments, and suggestions.

# How to Test This Pull Request
The folder hierarchy re-structuring should keep the CI working. Triiger CI to verify this pull request.
